resources/public/api/conf/P1.0/application.raml

```diff
@@ -9,10 +9,6 @@ mediaType: [ application/json, application/hal+json ]
 documentation:
  - title: Overview
    content: !include docs/overview.md
- - title: Versioning
```

Comment:
We will need versioning and error content in future so could we not leave these in and keep them as TODO?

Reply:
Resolved this :)